src/cobra/flux_analysis/variability.py

```diff
@@ -26,6 +26,29 @@
 configuration = Configuration()
 
 
+class _FakeProgress:
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *args):
+        pass
+
+    def add_task(self, *args, **kwargs):
+        return object()
+
+    def update(*args, **kwargs):
+        pass
+
+
+def _update_progress_bar(
```

Comment:
What is the point of this function? It just seems like an unnecessary level of indirection.